ttnn-visualizer/src/components/GoldenTensorComparisonIndicator.tsx

```diff
@@ -0,0 +1,26 @@
+import calculateOpPerformanceColor from '../functions/calculateOpPerformanceColor';
+// TODO: look at making this import path nicer (alias throw a linting error in .tsx files)
+import '../scss/components/GoldenTensorComparisonIndicator.scss';
+
+const GOLDEN_GLOBAL = [1, 0.9999, 0.9888, 1, 1, 0.8888];
```

Comment:
we ~probably~ definitely dont need those

Reply:
🔪